server/game/cards/05-UotE/MakeAnOpening.js

```diff
@@ -0,0 +1,34 @@
+const DrawCard = require('../../drawcard.js');
+
+class MakeAnOpening extends DrawCard {
+    setupCardAbilities(ability) { // eslint-disable-line no-unused-vars
+        this.action({
+            title: 'Give -X/-X to opposing character, where X is the difference between current honor dial bid values',
+            condition: context =>
+                this.game.isDuringConflict() &&
+                this.game.currentConflict.getNumberOfParticipantsFor(context.player) >= 1 &&
+                this.game.currentConflict.getNumberOfParticipantsFor(context.player.opponent) >= 1 &&
+                context.player.showBid &&
+                context.player.opponent.showBid &&
+                context.player.showBid !== context.player.opponent.showBid,
+            target: {
+                cardType: 'character',
+                controller: 'opponent',
+                cardCondition: card =>
+                    card.isParticipating() &&
+                    (card.militarySkill > 0 || card.politicalSkill > 0),

```

Comment:
It's legal to play this on a 0/0 character, as it's still considered a change in game state, so you shouldn't check for positive skills here